ios/GitPoint.xcodeproj/project.pbxproj

```diff
@@ -5,7 +5,6 @@
 	};
 	objectVersion = 46;
 	objects = {
-
```

Comment:
Don't push changes to this

Reply:
I didn't notice the change, my editor made it automatically. I hope now it's ok. Newbie mistake